docs/ske/01-kratix/10-configuring-ske/12-via-manifest.mdx

```diff
@@ -0,0 +1,178 @@
+SKE Operator. Locate the desired release of the SKE-Operator and apply its
+manifest with `kubectl`:
+
+```bash
+kubectl apply -f http://s3.eu-west-2.amazonaws.com/syntasso-enterprise-releases/ske-operator/<VERSION>/ske-operator-distribution.yaml
+```
+
+Applying the manifest will install the SKE Operator in the
+`kratix-platform-system` namespace, together with a Kratix Custom Resource
+Definition (CRD) that allows you to manage the Syntasso Kratix Enterprise installation.
+
+## Deploying SKE
+
+The SKE Operator includes a custom resource definition (CRD) for managing
+SKE. Only one instance of SKE can be created by the SKE Operator in a
+given cluster.
+
+To deploy SKE, create a file named `ske-instance.yaml` with the following contents:
+
+```yaml
+apiVersion: platform.syntasso.io/v1alpha1
+kind: Kratix
+metadata:
+  name: my-kratix
+spec:
+  version: v0.99.0 # The desired version of SKE
+```
+
+Apply the file with `kubectl`:
+
+```bash
+kubectl apply --filename ske-instance.yaml
+```
+
+The operator will then install SKE and its necessary configurations. For more
+information about the Kratix CRD, see the [SKE Operator](../ske-operator)
+documentation.
+
+## Upgrading SKE Operator
+
+To upgrade the SKE Operator, you can apply the new manifest directly:
+
+```bash
+kubectl apply -f http://s3.eu-west-2.amazonaws.com/syntasso-enterprise-releases/ske-operator/<NEW-VERSION>/ske-operator-distribution.yaml
+```
+
+## Upgrading SKE
+
+To upgrade the SKE instance, you can apply a new version of the Kratix CRD:
+
+```yaml
+apiVersion: platform.syntasso.io/v1alpha1
+kind: Kratix
+metadata:
+  name: my-kratix
+spec:
+  version: v0.100.0 # New desired version
+```
+
+### Upgrade checks
+
+The SKE Operator will perform [pre-upgrade checks](./ske-operator#pre-upgrade-checks)
+and [post-upgrade checks](./ske-operator#post-upgrade-checks) as part of the upgrade flow.
+
+To skip these checks, you can add the
+`ske.syntasso.io/skip-all-upgrade-checks` label when updating your `.spec.version`:
+
+```yaml
+apiVersion: platform.syntasso.io/v1alpha1
+kind: Kratix
+metadata:
+  name: my-kratix
+  #highlight-start
+  labels:
+    ske.syntasso.io/skip-all-upgrade-checks: "true"
+  #highlight-end
+spec:
+  version: v0.100.0 # New desired version
+```
+
+:::tip
+Skipping checks can be useful when you need to upgrade SKE while its current state is
+unhealthy.
+:::
+
+### Handling failures
+
+If any upgrade step fails, the SKE Operator will continue looping its reconciliation of
+the Kratix resource to pick up any manual fixes. See the section on [handling
+failures](../ske-operator#handling-failures) for more details.
+
+## Deleting SKE Operator
+
+To delete the SKE Operator (and any deployed SKE instance), run the following command:
+
+```bash
+kubectl delete -f http://s3.eu-west-2.amazonaws.com/syntasso-enterprise-releases/ske-operator/<VERSION>/ske-operator-distribution.yaml
+```
+
+:::warning
+
+This will delete the SKE Operator and any deployed SKE instance.
```

Comment:
Should we mention that this will delete all created resource requests and promises as well?